services/src/pro/server.rs

```diff
@@ -114,12 +115,9 @@ pub async fn start_pro_server(
 
     info!(
         "Starting server… {}",
-        format!(
-            "http://{}/",
-            web_config
-                .external_address
-                .unwrap_or(web_config.bind_address)
-        )
+        web_config
+            .external_address
+            .unwrap_or(Url::parse(&format!("http://{}/", web_config.bind_address))?)
```

Comment:
we need to enforce the trailing / here as well or not?

Reply:
it is checked that it is a valid socket address, so we can safely add the trailing slash

Reply:
c5b312d